packages/contract-wrappers/test/coordinator_wrapper_test.ts

```diff
@@ -0,0 +1,689 @@
+        });
+        it('should have coordinator server endpoints which respond to pings', async () => {
+            let result = await fetchAsync(`${coordinatorEndpoint}${coordinatorPort}/v1/ping`);
+            expect(result.status).to.be.equal(200);
+            expect(await result.text()).to.be.equal('pong');
+
+            result = await fetchAsync(`${coordinatorEndpoint}${anotherCoordinatorPort}/v1/ping`);
+            expect(result.status).to.be.equal(200);
+            expect(await result.text()).to.be.equal('pong');
+        });
+    });
+    // fill handling is the same for all fill methods so we can test them all through the fillOrder and batchFillOrders interfaces
+    describe('fill order(s)', () => {
+        describe('#fillOrderAsync', () => {
+            it('should fill a valid order', async () => {
+                txHash = await contractWrappers.coordinator.fillOrderAsync(
+                    signedOrder,
+                    takerTokenFillAmount,
+                    takerAddress,
+                );
+
+                await web3Wrapper.awaitTransactionSuccessAsync(txHash, constants.AWAIT_TRANSACTION_MINED_MS);
+            });
+        });
+        describe('#batchFillOrdersAsync', () => {
+            it('should fill a batch of valid orders', async () => {
+                const signedOrders = [signedOrder, anotherSignedOrder];
+                const takerAssetFillAmounts = [takerTokenFillAmount, takerTokenFillAmount];
+                txHash = await contractWrappers.coordinator.batchFillOrdersAsync(
+                    signedOrders,
+                    takerAssetFillAmounts,
+                    takerAddress,
+                );
+
+                await web3Wrapper.awaitTransactionSuccessAsync(txHash, constants.AWAIT_TRANSACTION_MINED_MS);
+            });
+            it('should fill a batch of orders with different feeRecipientAddresses with the same coordinator server', async () => {
+                const signedOrders = [signedOrder, anotherSignedOrder, signedOrderWithDifferentFeeRecipient];
+                const takerAssetFillAmounts = [takerTokenFillAmount, takerTokenFillAmount, takerTokenFillAmount];
+                txHash = await contractWrappers.coordinator.batchFillOrdersAsync(
+                    signedOrders,
+                    takerAssetFillAmounts,
+                    takerAddress,
+                );
+
+                await web3Wrapper.awaitTransactionSuccessAsync(txHash, constants.AWAIT_TRANSACTION_MINED_MS);
+            });
+            // coordinator-server, as currently implemented, shares a singleton database connection across
+            // all instantiations. Making the request to two different mocked server endpoints still hits the
+            // same database and fails because of the uniqueness constraint on transactions in the database.
+            it.skip('should fill a batch of orders with different feeRecipientAddresses with different coordinator servers', async () => {
```

Comment:
Will this still get fixed before we merge?

Reply:
No, this involves fixing the server side of things. I still think it's out of scope like we discussed. But will make a task to fix it after merging.